### PR TITLE
English Translation added to README file

### PR DESCRIPTION
**Pull request by khanaaziz1**
English Translation added to README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,4 +66,75 @@ orienter la recherche :
...
-je pense qu'elles vous aideront à débusquer les premiers pièges.
\ No newline at end of file
+je pense qu'elles vous aideront à débusquer les premiers pièges.
+
+
+# English
+
+## :family: Introduction
+
+I learned how to use the brakes on my downhill bike, and not just use the brake
```
Comment (olivierperez) [inline README.md]: Do we say "front brake" in english? To talk about the brake of the front wheel?

File: README.md [hunk 2]
```diff
@@ -66,4 +66,75 @@ orienter la recherche :
...
+to jump to the throat of my beloved colleagues, I prefer to feed on their mistakes. In this Kata
+we will see a summary of what you can meet in any project.
+
+The general idea is to leave with lots of good ideas to put in place tomorrow.
+
+**We will look at another developer's code and try to learn from it.**
+
+Keep in mind that the developer has (probably) not done on purpose to write form code
+puzzles. He produced code at some point, under specific circumstances, with
+the experience that was hers at that time. He'll do better next time, that's for sure.
```
Comment (olivierperez) [inline README.md]: "was hers" -> "was his/hers"?

File: README.md [hunk 3]
```diff
@@ -66,4 +66,75 @@ orienter la recherche :
...
+In short, we can strongly criticize a code while remaining in the respect of the one who wrote (hearts on him).
+
+## :information_source: Organization of a session
+
+### :books: Set up (10 minutes)
+
+Form groups of 2 or 3 developers.
+
+The organizer distributes to each group of participants one (or more) end (s) of code to
+analyze ([snippets] (/ kotlin / snippets /)). The idea is that each group has snippets
```
Comment (olivierperez) [inline README.md]: The link is broken

File: README.md [hunk 4]
```diff
@@ -66,4 +66,75 @@ orienter la recherche :
...
+different.
+
+Idea: _Print the snippets on sheets A5 or A6._
+
+### :mag_right: Running the kata (10-15 minutes by snippets)
```
Comment (olivierperez) [inline README.md]: by snippet (minus "s")

File: README.md [hunk 5]
```diff
@@ -66,4 +66,75 @@ orienter la recherche :
...
+
+Once the snippets are distributed, each group can begin their analysis. The analysis must be
+on the "cleanliness" of the code. The concept of cleanliness is rather relative to everyone, so we have to
+agree between participants in the same group on things that are wrong.
+
+Once the dirt has been identified, it is necessary to propose 1, 2 or 3 different alternatives of which at least
+better than the original snippet.
+
+### :mega: Pooling (5 minutes by snippets)
```
Comment (olivierperez) [inline README.md]: same

**Review by olivierperez — CHANGES_REQUESTED** [review]
Thanks a lot for translation :smile:
Can you?
- look at the comments
- move the english part to another file (`en/README.md`)
- Add a chapter in the main README "Other languages" that list translations

**Comment by khanaaziz1** [comment]
lemme go through it again :)

**Comment by khanaaziz1** [comment]
you can review now

File: README_OtherLanguages.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+In short, we can criticize a code while not hurting the feeling of writer.
+
+## :information_source: Organization of a session
+
+### :books: Set up (10 minutes)
+
+Form groups of 2 or 3 developers.
+
+The organizer will distributes code to each participanting group may be one or more to
+analyze ([snippets](/kotlin/snippets/)). The idea is that each group has snippet
```
Comment (olivierperez) [inline README_OtherLanguages.md]:
```suggestion
analyze ([snippets](/snippets/)). The idea is that each group has snippet
```

**Review by olivierperez — CHANGES_REQUESTED** [review]
One last change please :-)

File: README_OtherLanguages.md (hunk 6 above)
Comment (khanaaziz1) [reply README_OtherLanguages.md]: done

**Review by olivierperez — APPROVED** [review]

**Comment by olivierperez** [comment]
There are some conflicts, I can't merged.
Otherwise, it looks good to me.

**Comment by khanaaziz1** [comment]
let me check

**Comment by khanaaziz1** [comment]
is it ok if i refork and upload the file again

**Pull request closed without merging**